airflow/contrib/sensors/file_sensor.py

```diff
@@ -15,7 +15,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""This module is deprecated. Please use `airflow.sensors.filesystem`."""
+"""This module is deprecated. Please use :class:`airflow.sensors.filesystem`."""
```

Comment:
```suggestion
"""This module is deprecated. Please use :mod:`airflow.sensors.filesystem`."""
```

Since this is a module and not class, same for all other cases too

Reply:
Docs for it: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

The default language is `python` so you can use `:py:mod:` or just `:mod:`

Reply:
I didn't realize they were referencing modules. I'll use that!